steps/dsym-upload-to-crashlytics--no-cocoapods/0.0.1/step.yml

```diff
@@ -0,0 +1,35 @@
+title: Firebase dSym Upload (SPM installation)
+summary: |
+  Upload of dSym to Firebase Crashlytics with SPM installation
+description: |
+  This step will help you to upload dSym of your build to Firebase Crashlytics with SPM installation.
+website: https://github.com/hughmaurer/bitrise-step-dsym-upload-to-crashlytics--no-cocoapods
+source_code_url: https://github.com/hughmaurer/bitrise-step-dsym-upload-to-crashlytics--no-cocoapods
+support_url: https://github.com/hughmaurer/bitrise-step-dsym-upload-to-crashlytics--no-cocoapods/issues
+published_at: 2023-12-20T12:22:50.911776+01:00
+source:
+  git: https://github.com/hughmaurer/bitrise-step-dsym-upload-to-crashlytics--no-cocoapods.git
+  commit: 5613b9ade23a2f1472fe11eaa9e59da904d537d2
+type_tags:
```

Comment:
Not a blocking issue, but you might want to limit this step to certain project platforms: https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/developing-a-new-step.html#platforms